tests/integration/__init__.py

```diff
@@ -1,23 +1,167 @@
 import os
+from typing import Union
 
 from tests import TestBase
 
-
 class IntegrationTestBase(TestBase):
+    pass
```

Comment:
We probably don't even need this `IntegrationTestBase` anymore, should we just get rid of it and use TestBase?

Reply:
Removed the unnecessary class hierarchy.